src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltObservationTable.js

```diff
@@ -386,13 +408,65 @@ const FishBeltObservationTable = ({
             <Tr>
               <Th> </Th>
               <Th align="left" id="fish-name-label">
-                Fish Name <RequiredIndicator />
+                <LabelContainer>
+                  <div>
+                    Fish Name <RequiredIndicator />
+                  </div>
+                  {isHelperTextShowing && currentHelperTextLabel === 'fishName' ? (
+                    <ColumnHeaderToolTip
+                      helperText={language.tooltipText.fishName()}
+                      bottom="5.8em"
+                      paddingBottom="3.5em"
+                    />
+                  ) : null}
+                  <IconButton
+                    type="button"
+                    onClick={(event) => handleInfoIconClick(event, 'fishName')}
+                  >
+                    <IconInfo aria-label="info" />
+                  </IconButton>
+                </LabelContainer>
               </Th>
               <Th align="right" id="fish-size-label">
-                Size (cm) <RequiredIndicator />
+                <LabelContainer>
+                  <div>
+                    Size (cm) <RequiredIndicator />
+                  </div>
+                  {isHelperTextShowing && currentHelperTextLabel === 'fishSize' ? (
+                    <ColumnHeaderToolTip
+                      helperText={language.tooltipText.fishSize}
+                      bottom="5.8em"
```

Comment:
if I could figure out the exact clip path coordinates (instead of percentages) - I can avoid having to make some of this custom adjustments (bottom, padding-bottom)